app/api/gmail/bulk-operations/route.ts

```
@@ -0,0 +1,313 @@
+              { error: 'campaign_id required for add_to_campaign action' },
+              { status: 400 }
+            )
+          }
+
+          // Verify campaign ownership before proceeding
+          const { data: campaign, error: campaignError } = await supabase
+            .from('email_campaigns')
+            .select('id, user_id')
+            .eq('id', parameters.campaign_id)
+            .eq('user_id', userId)
+            .single()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)